quicklogic/common/toolchain_wrappers/ql_symbiflow

```diff
@@ -375,6 +360,17 @@ else
 	CURR_DIR="${PWD}/${SOURCE}"
 fi
 
+if [[ -f $SOURCE/$PART ]];then
+	CSV_PATH=`realpath $SOURCE/$PART`
+elif [[ -f $PART ]];then
+        CSV_PATH=`realpath $PART` 
+else
+      echo "invalid csv file/path"
+      exit 1
+fi	
+
+export PART=${CSV_PATH}
```

Comment:
Rather than exporting `PART` as the CSV file name I would leave the `PART` variable as-is and added a new one (eg. `PART_CSV_PATH`). Changing ,meaning of `PART` may make unification of these wrapper scripts with those used for other architectures more difficult.

Reply:
PART variable was introduced by Quicklogic and does not conflict with other architecture. I think going forward we will have the csv as input and not a fixed part name.

Reply:
Ok.